### ISSUE TITLE
Some AXML files start with another header

### ISSUE THREAD

**Issue by reox** [androguard/androguard#283]
It seems some AXML files (usually the AndroidManifest.xml), do not start with the usual AXML Header of `0x00080003` but with `0x00080503`. 
One solution is to allow this other header as well:
```
diff --git a/androguard/core/bytecodes/apk.py b/androguard/core/bytecodes/apk.py
index 1d4d780..458a76a 100644
--- a/androguard/core/bytecodes/apk.py
+++ b/androguard/core/bytecodes/apk.py
@@ -1222,7 +1222,7 @@ class AXMLParser(object):

         axml_file = unpack('<L', self.buff.read(4))[0]

-        if axml_file == CHUNK_AXML_FILE:
+        if axml_file == CHUNK_AXML_FILE or axml_file == 0x00080503:
             self.buff.read(4)

             header = ARSCHeader(self.buff)
```

Who has more knowledge of the AXML format and might answer the question with the different headers?
If you search for 0x00080503 on github you find nothing useful... It seems like nobody uses this constant? Where does it comes from?

**Comment by reox** [comment]
Some update on that: I scannen 50000 APKs and read the first four bytes of AndroidManifest.xml.
I found 319 files that had another header, namely:
```
      1  0x03570800
      1  0x88330800
      3  0x01000800
      3  0x03010800
      3  0x33330800
      4  0x00010800
      4  0x00020800
      5  0x00030800
      5  0x03020800
     11  0x02000800
    279  0x00000800
```

I did not tried to install them on a device (maybe I should check if those files are valid) but it looks like there are a lot of packers/obfuscators that change the header.
By the looks of it, the only thing that is always the same is the 0800.

I'll change the check, so such files are at least tried to read.

@androguard/external-contributors anyone has more information about the AXML file or this specific case? The documentation is really really sparse...
There are also other tickets regarding the AXML parser, does anyone has experience with this format?
